projects/storefrontlib/src/lib/user/store/selectors/user-orders.selectors.ts

```diff
@@ -12,7 +12,10 @@ export const getOrdersState: MemoizedSelector<
   (state: fromFeature.UserState) => state.orders
 );
 
-export const getOrders: MemoizedSelector<any, any> = createSelector(
+export const getOrders: MemoizedSelector<
+  any,
```

Comment:
I have no idea what that would be. `createSelector(...` returns a `MemoizedSelector<any, OrderHistoryList> `

Reply:
Take a look at `product-state.ts` and try with:
1. Create `export const USER_FEAUTRE = 'user';` in `projects/storefrontlib/src/lib/user/store/reducers/index.ts`
2. Create 
```
export interface StateWithUser {
  [USER_FEAUTRE]: UserState;
}
```
in the same file
3. Use `StateWithUser` instead of `any`

Reply:
Note the data user's structure before and after the change. It should be the same, without any new hierarchy level.
You can use redux dev tools for this.